tests/integration/Core/Repository/FieldType/ImageIntegrationTest.php

```diff
@@ -87,7 +95,12 @@ public function getSettingsSchema()
      */
     public function getValidFieldSettings()
```

Comment:
As we are touching those, maybe also here:
```suggestion
    public function getValidFieldSettings(): array
```

Reply:
Fixed: https://github.com/ibexa/core/pull/300/commits/32ecebef408bb7d4a7695059c7dd9e68c5cfc713

There are added ignore-next-lines tags because fileName is set correctly.